chart/otomi/templates/job.yaml

```diff
@@ -54,10 +52,8 @@ spec:
             - name: values-secret
               mountPath: /secret
         - name: map-values
-          image: {{ .Values.tasksImage.repository }}:{{ .Values.tasksImage.tag | default "latest" }}
-          {{- with .Values.image.pullPolicy }}
-          imagePullPolicy: {{ . }}
-          {{- end }}
+          image: otomi/tasks:latest
```

Comment:
Are you sure `latest` is stable? I would expect this is pinned to a semver, because I assume `latest` is unreliable.

Reply:
latest is latest release, so stable